src/graphql/schema/Playlist/playlist.js

```diff
@@ -1,18 +1,56 @@
 import { gql } from 'apollo-server-express';
+import Playlist from '../../../models/Playlist';
 
 // Type definitions for Playlists go here
 export const typeDefs = gql`
   extend type Query {
-    playlists: [Playlist!]!
+    getPlaylists: [Playlist]!
   }
+
+  extend type Mutation {
+    createPlaylist(name: String!, description: String): Playlist
```

Comment:
Does this pull a created playlist and show it on the user dashboard along with a album cover or something?

Reply:
This is the resolver where you call whatever mutation you need to call. Here, createPlaylist is called with the name and description parameters.  At the end of the call, you have to specify the type you expect back (in this case, Playlist) in return. What will come back is an empty playlist with nothing in it but the name of the playlist and the description if it is provided.